public/index.html

```diff
@@ -8,6 +8,7 @@
       manifest.json provides metadata used when your web app is added to the
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
```

Comment:
I think bootstrap dependencies should not be included in `index.html` because these are already installed in our project. You can check in the `package.json` file. I think we have installed `react-bootstrap` to use. So you can use those if you want some bootstrap. The way you can use `react-bootstrap`, you will find in the [docs](https://react-bootstrap.github.io/getting-started/introduction).